great_expectations/rule_based_profiler/data_assistant/data_assistant.py

```diff
@@ -22,6 +24,7 @@
     RuleBasedProfiler,
 )
 from great_expectations.rule_based_profiler.types import (
+    ALTAIR_CONFIGURATION,
```

Comment:
@NathanFarmer Based on the comment below, if the idea of moving plotting to the result object succeeds, then this `ALTAIR_CONFIGURATION` import statement would also move there.  Thanks.

Reply:
@alexsherstinsky Based on our conversations, I kept this here and clarified that this is the `ALTAIR_DEFAULT_CONFIGURATION` that can be overridden.